climpred/prediction.py

```diff
@@ -38,19 +49,88 @@ def compute_perfect_model(
         metric (str): `metric` name, see :py:func:`climpred.utils.get_metric_function`.
         comparison (str): `comparison` name, see
                           :py:func:`climpred.utils.get_comparison_function`.
+        dim (str or list): dimension to apply metric over. default: ['member', 'init']
         add_attrs (bool): write climpred compute args to attrs. default: True
+        ** metric_kwargs (dict): additional keywords to be passed to metric
```

Comment:
As in the bootstrap module, there should be a list here of which keywords are expected for which metrics. You only need to write it once then copy and paste in the different spots. Currently the user has to scroll through raw code to find out what keywords are accepted.

I guess they could also go to the metrics docs, but I think it would be good to have here for comprehensive documentation on one of our main functions.

Reply:
I will refer them to the metrics documentation.